person.rb

```diff
@@ -9,6 +9,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
```

Comment:
- [ ] In your `attr_accessor` declaration at the top of the class kindly add `:rentals` so that it can be set and get outside the class for creating a rental.

  ![yaseer-pers](https://user-images.githubusercontent.com/36542986/151344098-2a65f6d5-08a4-4e39-a09d-b18ef498b6ae.jpg)